### ISSUE TITLE
Synch with Wanikani getting overwritten 

### ISSUE THREAD

**Issue by heyimhowie** [davidsansome/tsurukame#626]
I just knocked out 200 reviews in Tsurukme only to have them overwritten by Wanikani after the synch. I tried a few more test runs, and it looks like Tsurukame is no longer synching progress up to Wanikani. Only from Wanikani down to Tsurukame.

Was pointed to this thread about WK disabling some api functionality. Pls help? 

https://community.wanikani.com/t/api-changes-get-all-reviews/61617

**Comment by davidsansome** [comment]
Hey!

Tsurukame doesn't use the API endpoint mentioned in that forum thread so it shouldn't be affected. Also syncing review progress seems to be working okay for me.

Your issue is probably something else. Can you upload your local database file to this bug? You can get it from Settings -> Export Local Database.

**Comment by heyimhowie** [comment]
Thanks for the quick response! Good to know it wasn’t the API mentioned. Here’s a link to my exported database. https://www.dropbox.com/sh/t10wa1d08sq6zfh/AAB6R4WXrP_7WKNTmnInVvx6a?dl=0

Thank you!

**Comment by ahmadmu** [comment]
yeah something stopped working yesterday, my stats are showing that I did no reviews today and yesterday:

![image](https://user-images.githubusercontent.com/11278706/235075430-4589d778-dc1c-4612-b547-7668e0c932a9.png)

**Comment by UInt2048** [comment]
@ahmadmu The issue is with Heatmap. Heatmap no longer has access to historical data. If you were to refresh Heatmap's data, you would find everything was erased. There's nothing we can do about this unfortunately.
